VimeoNetworking/VimeoNetworking/ErrorCode.swift

```diff
@@ -39,11 +39,11 @@ public enum VimeoErrorCode: Int
     case FacebookJoinMalformedToken = 2305
     case FacebookJoinDecryptFail = 2307
     case FacebookJoinTokenTooLong = 2308
-    case FacebookLogInNoToken = 2312
-    case FacebookLogInMissingProperty = 2310
-    case FacebookLogInMalformedToken = 2311
-    case FacebookLogInDecryptFail = 2313
-    case FacebookLogInTokenTooLong = 2314
```

Comment:
"login" is a noun and "log in" is a verb. These cases are nouns ("facebook login"), so they should be one word. If there are misspellings in other parts of the app we should correct them whenever we see them.